queries.sql

```diff
@@ -0,0 +1,11 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
* Kindly use the right query for this question instead of `'%mon%'` which will look for 'mon' anywhere in the string( animal name). you should use this instead of `'%mon'` this fetches animal names that ends with the string 'mon' 👍